packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx

```diff
@@ -72,7 +74,13 @@ export function DrawerNavigation({
         variant="persistent"
       >
         {!isMobile && (
-          <Stack alignItems="flex-start" sx={{ paddingLeft: '26px' }}>
+          <Stack
+            alignItems="flex-start"
+            sx={{ paddingLeft: '60px', cursor: 'pointer' }}
```

Comment:
Should it be 60px?

Looks a bit too much in comparison to design
<img width="412" alt="Screenshot 2024-10-25 at 18 47 08" src="https://github.com/user-attachments/assets/9489b86a-b4da-4df0-930c-08e5ce4724aa">
<img width="248" alt="Screenshot 2024-10-25 at 18 47 40" src="https://github.com/user-attachments/assets/301f5bda-2e24-401b-8545-de606a708ae7">

Reply:
@KacperKoza343 ☝🏼